### PR TITLE
Bugfix scale down navbar icons

### LINKED ISSUE #90: scale down navbar icons

### PR DESCRIPTION
**Pull request by franc-hoon**
Closes #90

### CODE REVIEW

**Review by thakurtushar02 — APPROVED** [review]
lgtm

File: src/stylesheets/components/navbar.scss [hunk 1]
```diff
@@ -33,8 +33,8 @@
 }
 
 .navbar__icon__group {
-	height: 70px;
-	width: 70px;
+	height: 55px;
```
Comment (max-gurr) [inline src/stylesheets/components/navbar.scss]: Might be better to make all these sizes in terms of vh. If you do the same values for both widthhs and heights, then everything will scale based on device height. It's more of a long-term solution than setting an absolute value

Comment (max-gurr) [reply src/stylesheets/components/navbar.scss]: Or even % values could work

File: src/stylesheets/variables.scss [hunk 2]
```diff
@@ -25,7 +25,7 @@ $button-shadow: 4px 4px 5px
 		$alpha: 0.25,
 	);
 
-$navbar-width: 90px;
+$navbar-width: 65px;
```
Comment (max-gurr) [inline src/stylesheets/variables.scss]: If you end up changing things to vh, then this will need to be changed as well to make sure everything scales

**Review by max-gurr — APPROVED** [review]
Looks good. There's lots of potential for regression issues with this PR, but that's kinda unavoidable.
I've suggested a change, but it's not required. I'll leave it up to you if you want to do it.

**Review by lucas2005gao — APPROVED** [review]
Looks good! I agree with @Angstylemon for changing some of the units to be more responsive, but don't stress on it too much if we are running short on time. Great work!

**Pull request merged**